software/base/src/test/java/brooklyn/entity/basic/lifecycle/StartStopSshDriverTest.java

```diff
@@ -113,10 +116,10 @@ public void testExecuteDoesNotLeaveRunningStreamGobblerThread() {
             @Override
             public void run() {
               List<ThreadInfo> currentThreads = getThreadsCalling(StreamGobbler.class);
-              List<Long> currentThreadIds = getThreadId(currentThreads);
+              Collection<Long> currentThreadIds = MutableSet.copyOf(getThreadId(currentThreads));
```

Comment:
Personal preference to declare as `Set<Long>`, given line 122 will compare it with a set.

But that's really minor, given that 3 lines earlier we can see it being instantiated!